Automattic/newspack-plugin
# Setup Wizard: Theme Style Screen

### All Submissions:






### Changes proposed in this Pull Request:

Add a screen to the Setup Wizard for selecting which theme (Newspack or one of the 5 Child Themes) to start with. Each theme can be previewed in the relatively recent `WebPreview` component and activated from there. The preview sites are:

- https://newspack.newspackstaging.com/
- https://nelson.newspackstaging.com/
- https://katharine.newspackstaging.com/
- https://joseph.newspackstaging.com/
- https://scott.newspackstaging.com/
- https://sacha.newspackstaging.com/

The PR introduces three new components: `ButtonGroup`, `StyleCard` and `StyleCardGroup`. The Webpack changes are here because this is the first case of raster assets being included in components. 

<img width="1662" alt="Screen Shot 2020-01-29 at 2 39 41 PM" src="https://user-images.githubusercontent.com/1477002/73391357-ca57a500-42a5-11ea-815e-e92f70749d31.png">

<img width="1662" alt="Screen Shot 2020-01-29 at 2 39 48 PM" src="https://user-images.githubusercontent.com/1477002/73391367-cf1c5900-42a5-11ea-891a-9cc5d34e7342.png">

### How to test the changes in this Pull Request:

1. After switching to this branch and building, navigate to the Setup Wizard (Settings->Newspack then click link at the bottom)
2. Advance to the sixth page: Theme
3. If you already have a Newspack theme activated, verify it is marked "Active."
4. Try Previewing and Activating each theme. Verify the preview looks good, and that the activation is successful. 

### Other information:
